tsconfig.json

```diff
@@ -8,7 +8,9 @@
     "skipLibCheck": true,
     "resolveJsonModule": true,
     "strict": true,
-    "module": "commonjs",
+    "removeComments": false,
```

Comment:
Waarom zou je comments niet willen removen?

Reply:
Or is this to keep the Webpack code splitting comments within import statements?

Reply:
Webpack adds comments for code-splitting to work. So we need to preserve them in the transpiled code